mbuild/formats/xyz.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+import mbuild as mb
+
+__all__ = ['read_xyz']
+
+
+def read_xyz(filename):
+    """Read an XYZ file.
+
+    Parameters
+    ----------
+    filename : str
+        Path of the output file
+
+    Returns
+    -------
+    compound : mb.Compound
+
+    Notes
+    -----
+    The XYZ file format neglects many important details, notably as bonds,
+    residues, and box information.
+
+    """
+
+    compound = mb.Compound()
+
+    with open(filename, 'r') as xyz_file:
+        n_atoms = int(xyz_file.readline())
+        next(xyz_file)
```

Comment:
this will need to be changed to xyz_file.readline() to pass tests on python2

Reply:
Ah, the python2 tests passed since there isn't any code that uses this reader yet. Also sorry for all the noise

Reply:
I should have probably waited to open this before I added the test